src/commands/new.rs

```diff
@@ -55,7 +97,7 @@ fn unpack_template(template: Vec<u8>, dir: &Utf8PathBuf) -> io::Result<()> {
     Ok(())
 }
 
-pub async fn new(path: Utf8PathBuf, name: Option<String>) -> Result<(), CliError> {
+pub async fn new(path: Utf8PathBuf, name: Option<String>, use_internet: bool) -> Result<(), CliError> {
```

Comment:
I think that `fetch_template` or similar is a better name than `use_internet` because it clearly shows what the internet is being used for.